### PR TITLE
DOC Refactor, reduce readme.md contents.

### PR DESCRIPTION
**Pull request by caspervdw**
As described in #208, this spread the contents of `readme.md` into the real documentation, and adds a reference to that in the `readme.md`.

Please review the PyAv part carefully, I am not sure if that is entirely correct. I haven't built the documents so it might need some debugging.

### CODE REVIEW

**Comment by caspervdw** [comment]
Could someone check up on this? I basically copy-pasted a lot of documentation so that `readme-md` doesn't get out of sync with the real documentation (where we want our users to go).

File: README.md [hunk 1]
```diff
@@ -6,26 +6,17 @@ pims: Python Image Sequence
...
-
-PIMS depends on [numpy](http://www.numpy.org) and [slicerator](https://github.com/soft-matter/slicerator).
-
-For basic image reading one of the following is required:
-
+Scientific video can be packaged in various ways: familiar video formats like
+.AVI and .MOV, folders full of numbered images, or "stacks" of TIFF images. Each
+of these requires a separate Python module. And, once loaded, they have
+different methods for **accessing individual images, looping through the images
+in bulk, or access a specific range**. PIMS can do all of these using a
```
Comment (nkeim) [inline README.md]: accessing

**Comment by nkeim** [comment]
Is there anywhere that someone can see an (approximately) complete list of supported formats? It would be helpful for prospective users, but also for existing users who'd like to refactor their workflows.

**Comment by caspervdw** [comment]
Agreed that this would be a good addition. We need to think of a way to describe 'formats' and maybe find a way to simplify the existing lists for [bioformats](https://www.openmicroscopy.org/site/support/bio-formats5.1/supported-formats.html) and [ffmpeg](https://ffmpeg.org/general.html#Supported-File-Formats_002c-Codecs-or-Features), which are probably the largest contributors.

**Comment by nkeim** [comment]
How about

Formats readable by pims include

- Directories or zipfiles of still images (most formats, including TIFF, JPEG, PNG, BMP), and TIFF stacks.
- [Microscope images supported by the Bio-formats project](https://www.openmicroscopy.org/site/support/bio-formats5.1/supported-formats.html), including Leica, Nikon, Olympus, and Zeiss formats (requires separate installation; see below).
- [Movie formats and codecs supported by ffmpeg](https://ffmpeg.org/general.html#Supported-File-Formats_002c-Codecs-or-Features), including AVI, QuickTime, and H.264 (may require separate installation; see below).
- CINE files from Vision Research cameras.
- SEQ files from NorPix StreamPix software.

A mention of 3D support would also be desirable, though I am unsure of how to work that in.

**Comment by caspervdw** [comment]
Good one, I force-pushed the change

**Comment by caspervdw** [comment]
I just pushed documentation on `FramesSequenceND` and its derived classes. As docwork is always ongoing work, this PR can be merged for my part if one of you has reviewed the text (especially the PyAv section).

File: README.md (hunk 1 above)
Comment (tacaswell) [inline README.md]: ^above comment is dealt with

File: README.md [hunk 2]
```diff
@@ -33,118 +24,10 @@ Depending on what file formats you want to read, you will also need:
...
-
-    pip install --upgrade http://github.com/soft-matter/pims/zipball/master
-
-Primary Contributors
---------------------
-* Daniel B. Allan
-* Thomas A. Caswell
-* Casper van der Wel
-
-Supporting Grant
```
Comment (tacaswell) [inline README.md]: The grant information should probably stay on the front page?

Comment (caspervdw) [inline README.md]: Agreed, I moved it back

**Comment by caspervdw** [comment]
Thanks for the review @tacaswell , I moved back the grant information and additionally added references to the multidimensional doc page in the Bioformats and ImageSequenceND pages.

One more thing, release notes, shall I just give them a separate page in the docs?

**Comment by tacaswell** [comment]
re release notes, yes

**Comment by caspervdw** [comment]
@nkeim, I have been messing up git on this branch, accidentally force-pushed a while ago so I lost the commit with the list of readable formats. I redid it, could you have one check on the latest commit?

**Comment by caspervdw** [comment]
@tacaswell I've just added the release notes.

**Comment by nkeim** [comment]
The current version looks right. Thanks!

**Comment by caspervdw** [comment]
@danielballan @tacaswell  Could someone merge this? Then we can go on with the release of 0.3.3. Would be nice to have it out soon.

**Comment by danielballan** [comment]
Sure thing.

**Comment by caspervdw** [comment]
Thanks! The travis build broke, probably due to a temporary server error for the bioformats download. I opened a PR to update bioformats anyway.

If you have time to build and push the docs then we can check if everything is OK, fix mistakes in the docs, and go on with the release.

**Pull request merged**